components/src/core/AppBar/AppBar.tsx

```diff
@@ -1,13 +1,6 @@
 import React, { useCallback, useEffect, useState } from 'react';
-import {
-    AppBar as MuiAppBar,
-    AppBarProps as MuiAppBarProps,
-    createStyles,
-    makeStyles,
-    Theme,
-    useMediaQuery,
-    useTheme,
-} from '@material-ui/core';
+import { createStyles, makeStyles, Theme, useMediaQuery, useTheme } from '@material-ui/core';
```

Comment:
You may want to look into the source code's `d.ts` in node_modules  to see from where these are exported.

```suggestion
import createStyles from '@material-ui/styles/createStyles';
import makeStyles from '@material-ui/styles/makeStyles';
import { Theme } from '@material-ui/core/styles/createMuiTheme';
import useTheme from '@material-ui/core/styles/useTheme';
import useMediaQuery from '@material-ui/core/useMediaQuery';
```

Reply:
Sure. Thanks for the feedback

Reply:
@huayunh Updated the PR with the suggested changes.